podpac/core/algorithm/stats.py

```diff
@@ -875,6 +875,11 @@ def _get_source_coordinates(self, requested_coordinates):
         native_time_mask = np.in1d(N, E)
 
         # use requested spatial coordinates and filtered available times
+        import pdb
+
+        pdb.set_trace()  # breakpoint e7271746 //
+
+        # WHAT?
```

Comment:
I'm guessing  you'd want to remove this breakpoint

Reply:
Thanks. This code is never called, and you can tell because it uses the **old** Coordinates spec.